src/v/cluster/rm_stm.cc

```diff
@@ -921,6 +964,12 @@ ss::future<tx_errc> rm_stm::do_abort_tx(
             // deadlock
             ssx::spawn_with_gate(
               _gate, [this, pid] { return try_abort_old_tx(pid); });
+            vlog(
+              _ctx_log.info,
```

Comment:
Why some of log message with info level, and some with trace. But looks like all of them about error during abort

Reply:
I use `trace` to log as much info as possible, `warn` when a user observes a problem, `info` when a may observe a problem and `error` when an internal invariant is broken.